boards/components/src/fxos8700.rs

```diff
@@ -0,0 +1,63 @@
+//! Components for the FXOS8700cq
+//!
+//! I2C Interface
+//!
+//! Usage
+//! -----
+//! ```rust
+//! let fxos8700 = components::fxos8700::Fxos8700Component::new(mux_i2c, PinId::AdB1_00.get_pin().as_ref().unwrap())
+//!    .finalize(());
+//!
+//! let ninedof = components::ninedof::NineDofComponent::new(board_kernel)
+//!    .finalize(components::ninedof_component_helper!(fxos8700));
+//! ```
+
+// Based on the component written for sam4l by:
+// Author: Philip Levis <pal@cs.stanford.edu>
+// Last modified: 6/03/2020
+
+#![allow(dead_code)] // Components are intended to be conditionally included
+#![allow(unused_imports)] // I2CDevice
```

Comment:
due to how generic components are included, there should be no need for the `#![allow(dead_code)]` that we used in the `imix_components`.

Also, I think it should be better to just remove the unused_import than to allow it.